rosco-manifests/src/main/java/com/netflix/spinnaker/rosco/manifests/helm/HelmBakeManifestService.java

```diff
@@ -45,7 +45,7 @@ public Artifact bake(HelmBakeManifestRequest request) {
     BakeStatus bakeStatus;
 
     try {
-      JobRequest jobRequest = new JobRequest(recipe.getCommand(), new ArrayList<>(), UUID.randomUUID().toString());
+      JobRequest jobRequest = new JobRequest(recipe.getCommand(), new ArrayList<>(), UUID.randomUUID().toString(), /* combineStdOutAndErr*/ false);
```

Comment:
Minor, but we usually don't add comments like this; the function definition is documentation of what the parameter does.